geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -185,10 +202,17 @@ public LocatorClusterManagementService(InternalConfigurationPersistenceService p
     }
 
     // persist configuration in cache config
-
-    configurationManager.add(config, groupName);
-    result.setStatus(StatusCode.OK,
-        "Successfully updated configuration for " + groupName + ".");
+    if (groups.size() == 1) {
+      String groupName = groups.iterator().next();
+      configurationManager.add(config, groupName);
+      result.setStatus(StatusCode.OK,
+          "Successfully updated configuration for " + groupName + ".");
+    } else {
+      for (String groupName : groups) {
+        configurationManager.add(config, groupName);
+      }
+      result.setStatus(StatusCode.OK, "Successfully updated configuration");
```

Comment:
I think this message should list all the groups it updated. Then you could get rid if the if size == 1 and just have all the code loop over the groups (even when it is just a single group).